src/test/java/org/broadinstitute/consent/http/resources/DataAccessRequestResourceTest.java

```diff
@@ -162,6 +165,99 @@ void testCreateDataAccessRequest() {
     assertEquals(Status.CREATED.getStatusCode(), response.getStatus());
   }
 
+  @NotNull
+  private User getRequestingUser() {
+    User requestingUser = new User(1, "requestor@test.com", "Requestor", new Date(), roles);
+    requestingUser.setInstitutionId(1);
+    return requestingUser;
+  }
+
+  @NotNull
+  private static Collaborator getCollaborator() {
+    Collaborator validCollaborator = new Collaborator();
+    validCollaborator.setEmail("collaborator@test.com");
+    return validCollaborator;
+  }
+
+  @NotNull
+  private static DataAccessRequest getDataAccessRequest(List<Collaborator> internalCollaborators) {
+    DataAccessRequestData data = new DataAccessRequestData();
+    data.setInternalCollaborators(internalCollaborators);
+    DataAccessRequest dar = new DataAccessRequest();
+    dar.setData(data);
+    return dar;
+  }
+
+  @Test
+  void testValidateInternalCollaboratorsNone() {
+    User requestingUser = getRequestingUser();
+    DataAccessRequest dar = getDataAccessRequest(Collections.emptyList());
+    initResource();
+    assertDoesNotThrow(() -> resource.validateInternalCollaborators(dar, requestingUser));
+  }
+
+  @Test
+  void testValidateInternalCollaboratorsValid() {
+    User requestingUser = getRequestingUser();
+    Collaborator validCollaborator = getCollaborator();
+    User collaboratorUser = new User(2, validCollaborator.getEmail(), "Collaborator", new Date(), roles);
+    collaboratorUser.setInstitutionId(requestingUser.getInstitutionId());
+    LibraryCard libraryCard = new LibraryCard();
+    libraryCard.setInstitutionId(requestingUser.getInstitutionId());
+    collaboratorUser.setLibraryCards(List.of(libraryCard));
+    DataAccessRequest dar = getDataAccessRequest(List.of(validCollaborator));
+    when(userService.findUserByEmail(validCollaborator.getEmail())).thenReturn(collaboratorUser);
+
+    initResource();
+    assertDoesNotThrow(() -> resource.validateInternalCollaborators(dar, requestingUser));
+  }
+
+  @Test
+  void testValidateInternalCollaboratorsDoesNotExist() {
+    User requestingUser = getRequestingUser();
+    Collaborator invalidCollaborator = getCollaborator();
+    DataAccessRequest dar = getDataAccessRequest(List.of(invalidCollaborator));
+    when(userService.findUserByEmail(invalidCollaborator.getEmail())).thenThrow(
+        new NotFoundException("Unable to find User with the provided email: " + invalidCollaborator.getEmail()));
+
+    initResource();
+    NotFoundException exception = assertThrows(NotFoundException.class, () ->
+        resource.validateInternalCollaborators(dar, requestingUser));
+    assertEquals(exception.getMessage(), "Unable to find User with the provided email: " + invalidCollaborator.getEmail());
```

Comment:
If we want to validate that the exception we throw above is caught here, I would make that explicit in the test.
```
    NotFoundException thrown = new NotFoundException);
    when(userService.findUserByEmail(invalidCollaborator.getEmail())).thenThrow(thrown);
   ...
    NotFoundException caught = assertThrows(NotFoundException.class, () ->
        resource.validateInternalCollaborators(dar, requestingUser));
    assertEquals(caught, thrown);
```
The message itself is unimportant and doesn't affect the code behavior, so adding one adds unnecessary complexity to the test.